doc/user/content/concepts/sinks.md

```diff
@@ -12,14 +12,15 @@ aliases:
 
 ## Overview
 
-Sinks are the inverse of sources and represent a connection to an external stream
```

Comment:
Replaced kafka-specific content with more generic content (same as in the server-results/sink page)